package.json

```diff
@@ -7,16 +7,13 @@
   "dist": "dist"
  },
  "scripts": {
-  "dist:svg": "node scripts/exportFromSketch.js src/sketch/ dist/svg/ web svg",
-  "dist:pdf": "node scripts/exportFromSketch.js src/sketch/ dist/pdf/ ios pdf",
-  "dist:js": "node scripts/generateConstants.js src/sketch dist/js/",
-  "dist:androidSvg": "node scripts/exportFromSketch.js src/sketch/ dist/svg/android/ android svg",
```

Comment:
I feel a little weird about removing Android scripts entirely while it's broken in Travis, but I guess version control makes it easy enough to bring it back when we're ready

Reply:
It looks like android exports were removed, but that export dist is now defined here in the export config: https://github.com/meetup/swarm-icons/pull/104/files#diff-1bc3842b3d2b4fea12503cb1ab35381cR21

The optimization step has been consolidated into one command (`yarn optimize`), but android optimization still happens.